src/sentry/issues/occurrence_consumer.py

```diff
@@ -319,6 +320,21 @@ def process_occurrence_message(
         txn.set_tag("result", "dropped_feature_disabled")
         return None
 
+    # Rate limit based on fingerprint
+    rate_limit_key = f"occurrence_rate_limit:{occurrence_data['fingerprint']}"
+    if ratelimiter.backend.is_limited(
```

Comment:
lets use `RedisSlidingWindowRateLimiter` instead of the default fixed window rate limiter, which will be a little better for this use case. (and can consider using Leaky bucket rate limiter in the future).

can see usage of it here https://github.com/getsentry/sentry/blob/eacc307d8dfdc4ac95fb5cd6a1d158c79dc35b74/src/sentry/utils/circuit_breaker2.py#L141